api/service/networkinfo/service.go

```diff
@@ -115,7 +115,18 @@ func (s *Service) DoService() {
 				return
 			}
 			if peer.ID != s.Host.GetP2PHost().ID() && len(peer.ID) > 0 {
-				utils.GetLogInstance().Info("Found Peer", "peer", peer.ID, "addr", peer.Addrs)
+				utils.GetLogInstance().Info("Found Peer", "peer", peer.ID, "addr", peer.Addrs, "my ID", s.Host.GetP2PHost().ID())
+				var wg sync.WaitGroup
+				wg.Add(1)
```

Comment:
let's do a Lock as group size is only 1 ?? it won't confuse developers. we can discuss if you disgree

Reply:
will do